challenges/ecosystem/02.md

```diff
@@ -51,21 +51,24 @@ npm install --dev eslint @typescript-eslint/eslint-plugin @typescript-eslint/par
 # same as `yarn add --dev eslint @typescript-eslint/eslint-plugin @typescript-eslint/parser`
 ```
 
-- [ ] Create a `.eslintrc.js` file at the root of your project with the following content: (you can use `touch .eslintrc.js` from the terminal).
+- [ ] Run `npx expo lint` to create a `.eslintrc.js` file at the root of your project.
+
+```console
+npx expo lint
+```
+
+- [ ] Update your `.eslintrc.js` file to add the following rules:
 
 ```javascript
 // .eslintrc.js
-
 module.exports = {
-  env: {
-    node: true
+  extends: ["expo", "react-native-wcandillon"],
```

Comment:
we do not use the `react-native-wcandillon` anymore on the bootcamp materials _—even if the rules are nice_
i want to stay simple with the most configurable way possible without adding too much magic.

Can you update `spacecraft` according to the new eslint rules from expo?